pkg/tools/tools.go

```diff
@@ -54,12 +54,20 @@ func ImageImportIntoClusterMulti(ctx context.Context, runtime runtimes.Runtime,
 		found := false
 		// Check if the current element is a file
 		if _, err := os.Stat(image); os.IsNotExist(err) {
+			// `runtimeImages` are returned without the 'docker.io/' prefix. So if `image` has such a prefix, we shall drop it.
+			localImageName := strings.TrimPrefix(image, "docker.io/")
+
+			// `runtimeImages` always contain a `:versionName` part. So if `image` doesn't, we shall add the default tag `:latest`.
+			if !strings.Contains(image, ":") {
```

Comment:
This will break, if you have images from a registry that is accessed via a specific port, e.g. `my.registry.org:9876/myrepo/myimage`, won't it?